src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.values(robot);
+  const values = Object.keys(robot);
+  const inverse = {};
+
+  for (const key of keys) {
+    if (keys.lastIndexOf(key) !== keys.indexOf(key)) {
+      return null;
+    }
+  }
+
+  for (let i = 0; i < keys.length; i++) {
+    inverse[keys[i]] = values[i];
+  }
```

Comment:
```suggestion
  for (let i = 0; i < keys.length; i++) {
   if (keys.indexOf(keys[i]) !== keys.lastIndexOf(keys[i])) {
      return null;
    }
  
    inverse[keys[i]] = values[i];
  }
```

i think u dont need 2 separate loops here.